docs/platforms/python/configuration/options.mdx

```diff
@@ -203,15 +203,15 @@ For many platform SDKs integrations can be configured alongside it. On some plat
 
 <ConfigKey name="integrations" />
 
-In some SDKs, the integrations are configured through this parameter on library initialization. For more information, please see our documentation for a specific integration.
+Fine-tune the integrations to enable. See [the integrations page](/platforms/python/integrations) for examples.
```

Comment:
`List of integrations to enable` is a bit misleading though. It could be understood to imply that this overrides the logic that auto-enables integrations. Maybe:

> List of integrations to enable in addition to auto-enabling integrations (link to integrations page). This setting can be used to override the default config options for a specific auto-enabling integration or to add an integration that is not auto-enabled.

Reply:
Yeah, I like your suggestion!